internal/testing/cluster_test.go

```diff
@@ -147,13 +147,25 @@ func TestUploadDownload(t *testing.T) {
 		}
 	}()
 
+	b := cluster.Bus
 	w := cluster.Worker
 	rs := defaultRedundancy
 
 	// add hosts
-	if _, err := cluster.AddHostsBlocking(int(rs.TotalShards)); err != nil {
+	if _, err := cluster.AddHostsBlocking(rs.TotalShards); err != nil {
+		t.Fatal(err)
+	}
+
+	// fund accounts
```

Comment:
TODO: should be removed or updated once EAs are refilled automatically by the autopilot